src/main/java/org/jitsi/jicofo/jigasi/JigasiDetector.java

```diff
@@ -96,11 +98,28 @@ protected void notifyInstanceOffline(Jid jid)
      * no Jigasis available currently.
      */
     public Jid selectJigasi()
```

Comment:
is this method still being used anywhere after this changes ?

Reply:
Yes, for transcriptions.